Cargo.toml

```diff
@@ -69,4 +69,4 @@ canonical_host = "0.4"
 [patch.crates-io]
 canonical = {git = "https://github.com/dusk-network/canonical", rev = "edf9efd"}
 canonical_derive = {git = "https://github.com/dusk-network/canonical", rev = "edf9efd"}
-canonical_host = {git = "https://github.com/dusk-network/canonical", rev = "edf9efd"}
\ No newline at end of file
+canonical_host = {git = "https://github.com/dusk-network/canonical", rev = "edf9efd"}
```

Comment:
We can do that only after we finish the blindbid update

Reply:
Blindbid update is done no? At least I understand that you considered it like this since you directly released the version `v0.5.0` without waiting for the docs nor anything similar 5 days ago.

Reply:
This is outside the scope of this PR

Reply:
I'm not following. What is the reason why we point to a specific commit of `canonical_host`? Why it can't be updated to the latest `canonical_host`?

Reply:
Ok, we agreed that this will be fixed in the following PRs, let's merge as is.